src/pages/dashboard/index.js

```diff
@@ -1,12 +1,77 @@
-import React from 'react'
+import { json } from 'd3-fetch'
+import { timeParse } from 'd3-time-format'
+
+import React, { useState, useEffect } from 'react'
 import Layout from '../../components/layout'
-import Map from './_Map'
-
-const DashboardPage = () => (
-  <Layout title="Visual Dashboard">
-    <p>Test</p>
-    <Map />
-  </Layout>
-)
+import AreaChart from './_AreaChart'
+// import Map from './_Map'
+
+const parseDate = timeParse('%Y%m%d')
+
+function calculateTotal(d) {
+  return d.positive + (d.negative || 0)
+}
+
+function transformUsDaily(data) {
+  return data
+    .map(d => {
+      const date = parseDate(d.date)
+      return [
+        {
+          date,
+          label: 'Total',
+          value: calculateTotal(d),
+        },
+        {
+          date,
+          label: 'Positive',
+          value: d.positive,
+        },
+      ]
+    })
+    .flat()
+}
+
+const DashboardPage = () => {
+  const [data, updateData] = useState([])
```

Comment:
the idiomatic way is `const [<value>, set<Value>] = useState([])` instead of `update`